topbar.js

```diff
@@ -799,16 +806,34 @@ function fixTopBar() {
     }
 }
 
+function fixWorkspaceIndicator() {
+    prefs.show_workspace_indicator ? menu.show() : menu.hide();
```

Comment:
This should just be an `if`, right?

Reply:
well, that's an inline if.  But yes, it's an if.

Reply:
I might have misunderstood your question(?).  It needs to be an `if-else`, since if we change the `show-workspace-indicator` option to false, we want it to `hide()`.

Hence the inline if(else) statement.